### ISSUE TITLE
Confusion about the code

### ISSUE THREAD

**Issue by Williamgithub1** [caiyuanhao1998/MST#20]
作者您好，我在您给出的代码里发现，在模拟集对Mask的处理当中，看到您给出的以下这种Phi_PhiPhiT类型，我想请教您一下这种Mask的是在什么情况下使用？期待您的回复。
`def generate_shift_masks(mask_path, batch_size):
    mask = sio.loadmat(mask_path + '/mask_3d_shift.mat')
    mask_3d_shift = mask['mask_3d_shift']
    mask_3d_shift = np.transpose(mask_3d_shift, [2, 0, 1])
    mask_3d_shift = torch.from_numpy(mask_3d_shift)
    [nC, H, W] = mask_3d_shift.shape
    Phi_batch = mask_3d_shift.expand([batch_size, nC, H, W])
    Phi_s_batch = torch.sum(Phi_batch**2,1)
    Phi_s_batch[Phi_s_batch==0] = 1
    return Phi_batch, Phi_s_batch`

**Comment by caiyuanhao1998** [comment]
这个是生成3D shifted mask才会使用的，你可以看看代码里面的调用关系来锁定具体使用的位置
